packages/build/src/log/messages/edge_manifest.js

```diff
@@ -0,0 +1,7 @@
+import { logMessage, logSubHeader } from '../logger.js'
+
+export const logEdgeManifest = ({ logs, manifest }) => {
+  logSubHeader(logs, 'Edge Functions Manifest')
+  logMessage(logs, JSON.stringify(manifest))
```

Comment:
nit: `JSON.stringify(manifest, null, 2)` would make this a bit easier to read.

Reply:
Oooo good point, I'll raise a separate PR to improve this

Reply:
I created a PR for this: https://github.com/netlify/build/pull/4375 - however, the snapshots don't seem to run on M1. The last PR Simon had to update the snapshots on his own machine and push them up. So might need some help updating the snapshots